docs(misc): remove some wrong references to the deprecated cacheableOperations option (#22483)
nrwl/nx: docs/shared/recipes/running-tasks/root-level-scripts.md

}
```

> To cache the `docs` target, you can set `cache: true` to the `docs` target shown in the `package.json` above and then your output would look like this:

Reviewer: ```suggestion
To cache the `docs` target, you can set `cache: true` on the `docs` target shown in the `package.json` above. Your output would then look as follows:
```
